### PR TITLE
Adding validation on application_periods and application_period_patte…

### PR DESCRIPTION
**Pull request by tchevily**
# Description

This PR fixes json validation for disruption

## Issue (optional)

Issue link: BOT-1004

## Pull Request type

This is a bug fix

### CODE REVIEW

File: chaos/formats.py [hunk 1]
```diff
@@ -338,12 +339,21 @@ def get_object_format(object_type):
...
+            'minItems': 1
         },
         'send_notifications': {'type': 'boolean'},
         'notification_date': {'type': ['string', 'null'], 'pattern': datetime_pattern}
     },
-    'required': ['severity', 'objects']
+    'required': ['severity', 'objects'],
+    'oneOf': [
+        {
+            'oneOf': [
```
Comment (pthegner) [inline chaos/formats.py]: Is the double `oneOf` necessary ?

Comment (tchevily) [reply chaos/formats.py]: yes

Comment (Netmisa) [reply chaos/formats.py]: @tchevily When I test your PR and I remove this double `oneOf` it works.
Could you explain why it's necessary please ?
Thx

Comment (tchevily) [reply chaos/formats.py]: venv/lib/python2.7/site-packages/jsonschema/tests/test_exceptions.py:131

Comment (pthegner) [reply chaos/formats.py]: i don't really understand this response, if you look on the line 88 (venv/lib/python2.7/site-packages/jsonschema/tests/test_exceptions.py:90) there is also just one oneOf.

**Comment by pthegner** [comment]
retest this please

**Pull request merged**